airflow/api_fastapi/execution_api/schemas.py

```diff
@@ -112,3 +112,10 @@ def ti_state_discriminator(v: dict[str, str] | BaseModel) -> str:
     ],
     Discriminator(ti_state_discriminator),
 ]
+
+
+class TIHeartbeatInfo(BaseModel):
+    """Schema for TaskInstance heartbeat endpoint."""
+
+    hostname: str
+    unixname: str
```

Comment:
Should we rename this to be more sensible in the protocol? (`username` say?)

I'd also be open to not capturing this at all -- I'm not sure what value it adds in keeping it.

Reply:
Yeah, can remove. What about pid?

Reply:
Replace unixname with pid in https://github.com/apache/airflow/pull/43722/commits/7c284c14b58275263b39d12981ff50b3670cbddd